<launch>

    <arg name="sim" default="true" />
    <arg name="ip" default="127.0.0.1" />
    <arg name="robot" default="ur5_cutter_modified" />
    <arg name="use_camera" default="true" />
    <arg name="use_servoing" default="false" />

    <group if="$(arg sim)">
        <node name="base_link_world_tf_publisher" pkg="tf" type="static_transform_publisher" args="0 0 0 0 0 0 1 world base_link 10"/>
    </group>

    <!-- Camera-related tfs -->
    <group if="$(arg use_camera)">
        <node name="map_camera_publisher" pkg="tf" type="static_transform_publisher" args="0.0266 0.0475 -0.10 0 0 0 1 or_ee_link camera_link 10" /> -->
        <include file="$(find realsense2_camera)/launch/rs_rgbd.launch" />
    </group>
    <group unless="$(arg use_camera)">
        <node name="map_camera_publisher" pkg="tf" type="static_transform_publisher" args="0.0266 0.0625 -0.10 -0.5 0.5 -0.5 0.5 or_ee_link camera_color_optical_frame 10" />
    </group>


    <include file="$(find sawyer_planner)/launch/launch_base.launch">
        <arg name="sim" value="$(arg sim)" />
        <arg name="ip" value="$(arg ip)" />
        <arg name="robot" value="$(arg robot)" />
        <arg name="global_camera_cloud" value="cloud_bottom_in" />
        <arg name="inhand_camera_cloud" value="cloud_inhand_filtered" />
        <arg name="base_frame" value="map" />
        <arg name="use_camera" value="$(arg use_camera)" />
    </include>

    <node if="$(arg use_servoing)" name="point_gravitation" pkg="sawyer_planner" type="point_gravitation.py" output="screen" />







</launch>